lvl_library/src/main/java/com/google/android/vending/licensing/APKExpansionPolicy.java

```diff
@@ -134,9 +139,9 @@ public void processServerResponse(int response,
             setRetryCount(mRetryCount + 1);
         }
 
+        // Update server policy data
+        Map<String, String> extras = decodeExtras(rawData);
         if (response == Policy.LICENSED) {
```

Comment:
mLicensingUrl (and associated shared preference) is never unset, which could cause setLicensingUrl to return stale data in the future. Should it get unset on a LICENSED response?

Reply:
Yeah, I think unsetting it on a LICENSED response is reasonable. Made both changes to the ServerManagedPolicy too.